snap/snapcraft.yaml

```diff
@@ -0,0 +1,68 @@
+name: imagecraft
+base: core22
+summary: Imagecraft is a craft tool used to create Ubuntu bootable images.
+description: |
+    Imagecraft is a craft tool used to create Ubuntu bootable images. It follows
+    the same principles as snapcraft, but is focused on creating bootable images
+    instead.
+adopt-info: imagecraft
+confinement: classic
+grade: devel
+license: GPL-3.0
+source-code: https://github.com/canonical/imagecraft
+
+apps:
+  imagecraft:
+    command: bin/python $SNAP/bin/imagecraft
```

Comment:
...should we just use `bin/python3` instead, explicitly?

Reply:
I think we can yes. This is another thing I left "as is" because it is done this way in every but one craft apps I looked at and was not sure how much of a convention this is. On this one I think you are right. I will fix it.